github/actions_artifacts.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2020 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// Artifact reprents a GitHub artifact.  Artifacts allow sharing
+// data between jobs in a workflow and provide storage for data
+// once a workflow is complete.
+//
+// GitHub API docs: https://developer.github.com/v3/actions/artifacts/
+type Artifact struct {
+	ID                 *int64  `json:"id,omitempty"`
+	NodeID             *string `json:"node_id,omitempty"`
+	Name               *string `json:"name,omitempty"`
+	SizeInBytes        *int64  `json:"size_in_bytes,omitempty"`
+	ArchiveDownloadURL *string `json:"archive_download_url,omitempty"`
+	Expired            *bool   `json:"expired,omitempty"`
```

Comment:
Can you please contact GitHub tech support at support@github.com and ask if they are actually sending the `expired` field as a `string` or as a JSON `boolean`? Their docs show a string, but I would think they should be a boolean.
If they say these are indeed strings like the example shows, we may need to change this field to a `*string` instead of a `*bool`.  You can report their reply back here, please.

Reply:
Sure thing.